src/map/clif.hpp

```diff
@@ -257,7 +257,7 @@ enum broadcast_flags : uint8_t {
 	BC_DEFAULT		= BC_ALL|BC_PC|BC_YELLOW
 };
 
-enum emotion_type {
+enum emotion_type : uint8 {
```

Comment:
```suggestion
enum e_emotion_type : uint8 {
```
Can you please rename the references too?

Reply:
i can't find any ref to it '-'